src/main/java/org/cactoos/scalar/ItemAt.java

```diff
@@ -92,8 +93,8 @@ public ItemAt(
      */
     public ItemAt(
         final int position,
-        final Func<Iterable<T>, T> fallback,
-        final Iterable<T> iterable
+        final Func<? super Iterable<? super T>, ? extends T> fallback,
```

Comment:
@rocket-3 I think this should be `Func<? super Iterable<? extends T>, ? extends T>`, i.e., with `extends` for the `Iterable` type parameter.